### DISCUSSION TITLE
Questions on what was confusing that could use explanation on the page or here

### DISCUSSION THREAD

**Discussion by JustinGOSSES** [softwareunderground/open_geosciene_code_projects_viz#13] [Q&A]
Questions on what was confusing that could use explanation on the page or here in discussions

**Comment by RichardScottOZ** [comment]
Organisations on the add repos list - all repositories from those go into the visualisations?

**Reply by JustinGOSSES** [reply]
Yes

I'll try to add some text about this on the page
